### PR TITLE
Update README to inform about release tags

### PR DESCRIPTION
**Pull request by DiogoAndre**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,3 +21,9 @@ If you have an issues specific to the [Realm Mobile Database](https://realm.io/p
 * [Objective-C & Swift](https://github.com/realm/realm-cocoa)
 * [React Native](https://github.com/realm/realm-js)
 * [Xamarin](https://github.com/realm/realm-dotnet)
+
+## Releases
+
+Beta versions will be coming out frequently during the Beta period, and a new relase tag will be added to this repository when it does. You can Watch this repo to get notified when a new Beta version is out.
```
Comment (alebsack) [inline README.md]: This should be "a new release tag will be added to this repository when they do"

**Comment by DiogoAndre** [comment]
Thanks for your review @alebsack! Will make the corrections shortly.

File: README.md [hunk 2]
```diff
@@ -21,3 +21,9 @@ If you have an issues specific to the [Realm Mobile Database](https://realm.io/p
 * [Objective-C & Swift](https://github.com/realm/realm-cocoa)
 * [React Native](https://github.com/realm/realm-js)
 * [Xamarin](https://github.com/realm/realm-dotnet)
+
+## Releases
+
+Beta versions will be coming out frequently during the Beta period, and a new release tag will be added to this repository when they do. You can Watch this repo to get notified when a new Beta version is out.
+
+Check the [Releases Page](https://github.com/realm/realm-mobile-platform/releases) to see the list of latest vesions and the changes introduced with each one.
```
Comment (alebsack) [inline README.md]: Almost there...  "versions" is misspelled.

Comment (DiogoAndre) [inline README.md]: uhh.. sorry. Fixing it.

**Pull request merged**